nodepool/nl01.sjc1.vexxhost.zuul.ansible.com.yaml

```diff
@@ -344,9 +344,11 @@ providers:
               - Private Network (Floating Public)
               - Public Internet
           - name: iosxrv-7.0.2
-            flavor-name: s1.large
+            flavor-name: s1.xlarge
             cloud-image: iosxrv-7.0.2-20211703
             host-key-checking: false
+            boot-from-volume: true
+            volume-size: 150
```

Comment:
why do we need 150 GB here? Also, why are we booting from volume, we shouldn't need to do this.